packages/react-native-bridge/android/settings.gradle

```diff
@@ -1,5 +1,52 @@
-rootProject.name = '@wordpress/react-native-bridge'
+pluginManagement {
+  plugins {
+    id 'com.jfrog.bintray' version '1.8.5'
+  }
+}
 
+rootProject.name = '@wordpress_react-native-bridge'
+
+include ':react-native-bridge'
 include ':@wordpress_react-native-aztec'
 project(':@wordpress_react-native-aztec').projectDir = new File(rootProject.projectDir, '../../react-native-aztec/android')
 
+def publishReactNativeBridgeVersion = properties["publishReactNativeBridgeVersion"]
+gradle.ext.willPublishReactNativeBridgeBinary = publishReactNativeBridgeVersion?.trim() as boolean
+
+// Only add these modules/projects if we are not building to publish a binary.
+if (!gradle.ext.willPublishReactNativeBridgeBinary) {
+    /*
+    IMPORTANT: DO NOT add hermes modules if we are building a binary to publish.
+    If these modules are added, it'll cause the `com.facebook.hermes.BuildConfig` file to be
+    included in the binary which will then result in a duplicate class error.
+    This seems to happen even if these modules are not added as a dependency.
+    */
+    include ':hermes-debug'
+    include ':hermes-release'
+
+    assert file("../../../node_modules").exists() : "node_modules folder is missing. Please ensure that you run `npm install` from the root Gutenberg project"
```

Comment:
Nitpick.

```suggestion
    assert file("../../../node_modules").exists() : "node_modules folder is missing. Please run `npm install` from the root Gutenberg project"
```

Reply:
Addressed in https://github.com/oguzkocer/gutenberg/commit/4c67a6c540ba2fda7776c9e3b93d7f8381bf80d4